pkg/subscraping/sources/crtsh/crtsh.go

```
@@ -72,6 +71,20 @@ func (s *Source) getSubdomainsFromSQL(ctx context.Context, domain string, sessio
 		}
 	}()
 
+	conn, err := db.Conn(ctx)
+	if err != nil {
+		results <- subscraping.Result{Source: s.Name(), Type: subscraping.Error, Error: err}
+		s.errors++
+		return 0
+	}
+	defer conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)